### DISCUSSION TITLE
More Languages

### DISCUSSION THREAD

**Discussion by jonidimo** [kudrykv/latex-yearly-planner#21] [Ideas]
I wanted to change the language and to add a spanish version. I downloaded the source code but I haven't find where to modify this. There is any chance you put this in more languages? I am fully available to help you. Thank you.

**Comment by kudrykv** [comment]
Hi @jonidimo!

Thank you for the suggestion!

Adding localization would be tricky. It is easy to update strings themselves in the template, like "notes," etc. E.g., [here](https://github.com/kudrykv/latex-yearly-planner/blob/919ffce529ee732553a6badd17448a8bafdc3ad5/tpls/_common_05_daily.tpl#L16) is the line on daily pages that has a few strings. However, headers and date-related fields assume English (yup, that's something I didn't quite think through ¯\\\_(ツ)\_/¯). For example, [here](https://github.com/kudrykv/latex-yearly-planner/blob/919ffce529ee732553a6badd17448a8bafdc3ad5/app/components/cal/day.go#L70-L76) is the part of the code that generates a header for daily pages for the breadcrumb template. It implies English and uses constants from Go, which are in English.

I need to think about how to implement localization best. Of course, I'll be glad to hear suggestions if you'll have any!

**Comment by terodactilo21** [comment]
Sólo me registré para comentar que también me gustaría tenerlo en español. Mi inglés no es bueno. ¿Cómo podría ayudar? Muchas gracias por su trabajo.

**Reply by kudrykv** [reply]
Hi @terodactilo21!
I'll look into localizations after I complete the refactor. I'll keep a note of the ask, though!

**Comment by bracketouverte** [comment]
1st of all thanks for your great job 👍 

Even if I'm not able to provide code to achieve that, I'm fully available if you need help getting documentation to translate in french / getting french dates format.

**Comment by terodactilo21** [comment]


**Reply by kudrykv** [reply]
I'm sorry, no. This year was, hm, busy
